src/groundwaterdata.jl

```diff
@@ -0,0 +1,47 @@
+using DataFrames
+using RData
+
+
+if isfile(datapath("cache/gwmodel/dfgw$suffix.csv"))
+    println("Loading saved groundwater model...")
+    dfgw = readtable(datapath("cache/gwmodel/dfgw$suffix.csv"));
+    lateralconductivity = convert(Array, readtable(datapath("cache/gwmodel/lateralconductivity$suffix.csv")));
+    aquiferconnexion = convert(Array, readtable(datapath("cache/gwmodel/aquiferconnexion$suffix.csv")));
+
+else
+    if config["dataset"] == "counties"
+        dfgw = readtable(datapath("gwmodel/dfgw.csv"));
+        lateralconductivity = deserialize(open(datapath("gwmodel/lateralconductivity.jld"), "r"));
+        aquiferconnexion = deserialize(open(datapath("gwmodel/aquiferconnexion.jld"), "r"));
+        
+        if config["filterstate"] != nothing
+            println("Generating regionnal groundwater model...")
+	    vstates = round(Int64, floor(dfgw[:fips] ./ 1000));
+	    subfips = find(vstates .== parse(Int64, get(config,"filterstate", nothing)));
+	
+            dfgw = dfgw[subfips,:];
+
+  	    lateralconductivity = lateralconductivity[subfips,subfips];
+  	    aquiferconnexion = aquiferconnexion[subfips,subfips];
+
+            # Save groundwater model
+            writetable(datapath("cache/gwmodel/dfgw$suffix.csv"), dfgw)
+            writetable(datapath("cache/gwmodel/lateralconductivity$suffix.csv"), convert(DataFrame, lateralconductivity))
+            writetable(datapath("cache/gwmodel/aquiferconnexion$suffix.csv"), convert(DataFrame, aquiferconnexion))
```

Comment:
Is there a significant benefit from caching these?  It seems like either way we're just reading in a data file.

Reply:
It is only for when the model is run at a county-scale for a specific state that a partial gwmodel would be created ... it makes the simulation of groundwater faster ... I can delete it if you find it superfluous